test/e2e/smoketest.go

```diff
@@ -69,15 +69,12 @@ func executeSmokeTest(apiTracesEndpoint, collectorEndpoint string) {
 	closer.Close()
 
 	err = wait.Poll(retryInterval, timeout, func() (done bool, err error) {
-		c := http.Client{Timeout: time.Second}
+		c := http.Client{Timeout: 3 * time.Second}
 		req, err := http.NewRequest(http.MethodGet, apiTracesEndpoint+"?service="+serviceName, nil)
-		if err != nil {
-			return false, err
-		}
+		require.NoError(t, err)
```

Comment:
Just to clarify - I assume this means you are not expecting errors from the issuing of the request? so if it fails you want the polling to terminate immediately?

Reply:
Correct.  I think returning error cause polling to terminate anyway.

Reply:
Good point - it is probably the next one that is masking the error